core/app/helpers/spree/mail_helper.rb

```diff
@@ -11,14 +11,23 @@ def name_for(order)
       order.name || Spree.t('customer')
     end
 
+    def logo_store
+      @order&.store || current_store
```

Comment:
according to method name, should return `logo`

```suggestion
      @order&.store.logo || current_store.logo
```

Reply:
This is supposed to return a store which logo should be used.

Reply:
in that case, name of this method is little misleading, could we change that ?